upup/pkg/fi/cloudup/awstasks/launchtemplate.go

```diff
@@ -74,6 +74,10 @@ type LaunchTemplate struct {
 	Tenancy *string
 	// UserData is the user data configuration
 	UserData *fi.ResourceHolder
+	// HTTPTokens is the state of token usage for your instance metadata requests.
+	HTTPTokens *string
+	// HTTPPutResponseHopLimit is the desired HTTP PUT response hop limit for instance metadata requests.
+	HTTPPutResponseHopLimit *int64
```

Comment:
One more tiny nit. These should be moved a bit up to be alphabetically ordered. It looks a bit odd when creating the cluster. same in `launchconfiguration.go`.